Sources/ApolloCodegenLib/Frontend/dist/ApolloCodegenFrontend.bundle.js

```diff
@@ -1 +1 @@
pe)):we(e)?new GraphQLNonNull(E(e.ofType)):T(e)}function T(e){return h[e.name]}function N(e){return At(e)||pt(e)?e:_e(e)?function(e){var t;const n=e.toConfig(),r=null!==(t=l[n.name])&&void 0!==t?t:[];let i=n.specifiedByURL;for(const e of r){var o;i=null!==(o=Pn(e))&&void 0!==o?o:i}return new GraphQLScalarType({...n,specifiedByURL:i,extensionASTNodes:n.extensionASTNodes.concat(r)})}(e):be(e)?function(e){var t;const n=e.toConfig(),r=null!==(t=l[n.name])&&void 0!==t?t:[];return new GraphQLObjectType({...n,interfaces:()=>[...e.getInterfaces().map(T),...w(r)],fields:()=>({...Z(n.fields,I),...S(r)}),extensionASTNodes:n.extensionASTNodes.concat(r)})}(e):Oe(e)?function(e){var t;const n=e.toConfig(),r=null!==(t=l[n.name])&&void 0!==t?t:[];return new GraphQLInterfaceType({...n,interfaces:()=>[...e.getInterfaces().map(T),...w(r)],fields:()=>({...Z(n.fields,I),...S(r)}),extensionASTNodes:n.extensionASTNodes.concat(r)})}(e):Se(e)?function(e){var t;const n=e.toConfig(),r=null!==(t=l[n.name])&&void 0!==t?t:[];return new GraphQLUnionType({...n,types:()=>[...e.getTypes().map(T),...x(r)],extensionASTNodes:n.extensionASTNodes.concat(r)})}(e):Le(e)?function(e){var t;const n=e.toConfig(),r=null!==(t=l[e.name])&&void 0!==t?t:[];return new GraphQLEnumType({...n,values:{...n.values,...D(r)},extensionASTNodes:n.extensionASTNodes.concat(r)})}(e):Ae(e)?function(e){var t;const n=e.toConfig(),r=null!==(t=l[n.name])&&void 0!==t?t:[];return new GraphQLInputObjectType({...n,fields:()=>({...Z(n.fields,(e=>({...e,type:E(e.type)}))),...A(r)}),extensionASTNodes:n.extensionASTNodes.concat(r)})}(e):void n(!1,"Unexpected type: "+m(e))}function I(e){return{...e,type:E(e.type),args:e.args&&Z(e.args,g)}}function g(e){return{...e,type:E(e.type)}}function _(e){const t={};for(const r of e){var n;const e=null!==(n=r.operationTypes)&&void 0!==n?n:[];for(const n of e)t[n.operation]=b(n.type)}return t}function b(e){var t;const n=e.name.value,r=null!==(t=Vn[n])&&void 0!==t?t:h[n];if(void 0===r)throw new Error(`Unknown type: "${n}".`);return r}function O(e){return e.kind===u.LIST_TYPE?new GraphQLList(O(e.type)):e.kind===u.NON_NULL_TYPE?new GraphQLNonNull(O(e.type)):b(e)}function S(e){const t=Object.create(null);for(const i of e){var n;const e=null!==(n=i.fields)&&void 0!==n?n:[];for(const n of e){var r;t[n.name.value]={type:O(n.type),description:null===(r=n.description)||void 0===r?void 0:r.value,args:L(n.arguments),deprecationReason:Mn(n),astNode:n}}}return t}function L(e){const t=null!=e?e:[],n=Object.create(null);for(const e of t){var r;const t=O(e.type);n[e.name.value]={type:t,description:null===(r=e.description)||void 0===r?void 0:r.value,defaultValue:rn(e.defaultValue,t),deprecationReason:Mn(e),astNode:e}}return n}function A(e){const t=Object.create(null);for(const i of e){var n;const e=null!==(n=i.fields)&&void 0!==n?n:[];for(const n of e){var r;const e=O(n.type);t[n.name.value]={type:e,description:null===(r=n.description)||void 0===r?void 0:r.value,defaultValue:rn(n.defaultValue,e),deprec
```

Comment:
What changed in the JS bundle?

Reply:
I'm an idiot! It was right there in the `index.js` file - `referencedTypes.add(typeCondition)`. 🤦🏻

Reply:
lol no worries! It’s just a bug fix. The commit messages for the commits that make the changes explain why.